scripts/docs_version_pragma_check.sh

```diff
@@ -108,7 +108,7 @@ function findMinimalVersion()
     do
         if versionGreater "$ver" "$pragmaVersion"
         then
-            minVersion="$ver"
+            version="$ver"
```

Comment:
And this resulted in the same error if the version pragma as using `>=` or `^` rather than `>`.